src/04_printing.py

```diff
@@ -11,7 +11,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print("x is %s,y is %s, z is %s" % (x, y, z))
 # Use the 'format' string method to print the same thing
-
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print('x is {0}, y is {1}, z is {2}'.format(x, y, z))
+# Finally, print the same thing using an f-string
+print(f'x is {x},y is {y}, z is {z!r}')
```

Comment:
Great work with these three. Which implementation is your fav? 
Cool use of the `!r` here. Its a clear example of where `!r` applies `repr()` which converts the value before its formatted thereby leaving the quotes in the final output. NEAT!